src/org/thoughtcrime/securesms/sms/MessageSender.java

```diff
@@ -73,11 +74,14 @@ public static long send(final Context context,
       allocatedThreadId = threadId;
     }
 
+    //dont insert if reaction
```

Comment:
I'm not sure what this comment is supposed to say, it does not seem related to the code.